manifests/state-ofed-driver/0050_ofed-driver-ds.yaml

```
@@ -135,6 +143,47 @@ spec:
             initialDelaySeconds: {{ .CrSpec.ReadinessProbe.InitialDelaySeconds }}
             failureThreshold: 1
             periodSeconds: {{ .CrSpec.ReadinessProbe.PeriodSeconds }}
+        {{- if .RuntimeSpec.UseDtk }}
+        - image: {{ .RuntimeSpec.DtkImageName }}
+          imagePullPolicy: IfNotPresent
+          name: openshift-driver-toolkit-ctr
+          command: [bash, -xc]
+          args: ["until [ -f /mnt/shared-nvidia-driver-toolkit/dir_prepared ]; do echo  Waiting for nvidia-driver-ctr container to prepare the shared directory ...; sleep 10; done; exec /mnt/shared-nvidia-driver-toolkit/ocp_dtk_entrypoint dtk-build-driver"]
```

nit: maybe something like this is easier to read

```suggestion
          command: 
          - "/usr/bin/env bash"
          - "-c"
          - | 
          until [ -f /mnt/shared-nvidia-driver-toolkit/dir_prepared ]
          do 
            echo  "Waiting for nvidia-driver-ctr container to prepare the shared directory..."
            sleep 10
          done
          exec /mnt/shared-nvidia-driver-toolkit/ocp_dtk_entrypoint dtk-build-driver"]
```

---

Btw, which container is `nvidia-driver-ctr`?

---

This API is still in discussion, I will try this way once concluded.

---

Done